packages/appkit/tests/appkit.test.ts

```diff
@@ -869,6 +869,15 @@ describe('Base', () => {
     })
 
     it('should  sync balance correctly', async () => {
+      vi.spyOn(ChainController, 'state', 'get').mockReturnValue({
```

Comment:
We'd need to rebase to main as AppKit unit tests has been refactored. We could handle ths PRs cases in `balance.test.ts`

+ we shouldn't mock controllers, we should;
- Initialize AppKit
- Pass mock user to `syncAccount` (if we want to make user connected)
- Then to tests